go/sqlmetrics/connection.go

```diff
@@ -51,8 +51,29 @@ func loadConnInfo(conn *proxy.Conn) *connInfo {
 	return info.(*connInfo)
 }
 
-func parseDSN(dsn string) (*connInfo, error) {
-	u, err := dburl.Parse(dsn)
+// Note: the DB Url library expects a scheme in the DSN for
```

Comment:
This is not a correct way to handle this. Sql package supports magnitudes of queriersz we cannot switch case for all. Let the expected dan contain the scheme.